backend/tracim_backend/lib/mail_notifier/sender.py

```diff
@@ -85,10 +86,15 @@ def connect(self):
                     )
                     log = 'SMTP login result: {}'
                     logger.debug(self, log.format(login_res))
+                    logger.info(self, 'SMTP Connection OK')
+                except smtplib.SMTPAuthenticationError as e:
+                    log = 'SMTP auth params combinaison (login/password) ' \
+                          'are uncorrect'
```

Comment:
you really sure? What if the server reply is something like: "I do not access authentication for the next 10 minutes"? Do not interpret: log the facts.

Reply:
Here you should log "AUTH error: {}".format(e)